packages/app/src/cli/utilities/extensions/locales-configuration.test.ts

```diff
@@ -101,7 +101,7 @@ describe('loadLocalesConfig', () => {
       const es = joinPath(localesPath, 'es.json')
 
       await mkdir(localesPath)
-      const bigArray = JSON.stringify(new Array(3000).fill('a'))
+      const bigArray = JSON.stringify(new Array(4000).fill('a'))
```

Comment:
Did we have to increase this? We are only increasing the per file size limit, so was only expecting to see the above test changed ("Throws if one locale is too big") and not this one ("Throws if bundle is too big")?